data_diff/diff_tables.py

```diff
@@ -120,25 +137,45 @@ def _get_stats(self) -> DiffStats:
         table2_count = self.info_tree.info.rowcounts[2]
         unchanged = table1_count - diff_by_sign["-"] - diff_by_sign["!"]
         diff_percent = 1 - unchanged / max(table1_count, table2_count)
+        if is_dbt:
+            return DiffStats(diff_by_sign, table1_count, table2_count, unchanged, diff_percent, extra_column_diffs)
+        else:
+            return DiffStats(diff_by_sign, table1_count, table2_count, unchanged, diff_percent, None)
 
-        return DiffStats(diff_by_sign, table1_count, table2_count, unchanged, diff_percent)
-
-    def get_stats_string(self):
+    def get_stats_string(self, is_dbt:bool = False):
 
         diff_stats = self._get_stats()
-        string_output = ""
-        string_output += f"{diff_stats.table1_count} rows in table A\n"
-        string_output += f"{diff_stats.table2_count} rows in table B\n"
-        string_output += f"{diff_stats.diff_by_sign['-']} rows exclusive to table A (not present in B)\n"
-        string_output += f"{diff_stats.diff_by_sign['+']} rows exclusive to table B (not present in A)\n"
-        string_output += f"{diff_stats.diff_by_sign['!']} rows updated\n"
-        string_output += f"{diff_stats.unchanged} rows unchanged\n"
-        string_output += f"{100*diff_stats.diff_percent:.2f}% difference score\n"
-
-        if self.stats:
-            string_output += "\nExtra-Info:\n"
-            for k, v in sorted(self.stats.items()):
-                string_output += f"  {k} = {v}\n"
+        if is_dbt:
+            diff_stats = self._get_stats_dbt()
```

Comment:
no need for this right?

Reply:
@dlawin I think I actually need to change it to `diff_stats = self._get_stats(is_dbt)`, right? And then remove the redundant declaration of `diff_stats` like you suggested? See latest version.

Reply:
oh yeah true